### PR TITLE
Culture Convos 7 through 9 from Marksman

### PR DESCRIPTION
**Pull request by MasterOfGrey**
**Content (Missions)**

## Summary
These three conversations come to you courtesy of @TheMarksman-ES producing drafts to turn into a PR.

For concept criticism please direct your comments at him
For implementation including the mission structure, source locations or miscellaneous grammar fixes that'll be me.

### CODE REVIEW

File: data/human/culture conversations.txt [hunk 1]
```diff
@@ -80,11 +80,39 @@ outfit "Mug"
...
+				`	(Leave them be.)`
+					goto leave
+			`	You call out a warning and most of the men pause to look at you, the hull plate swinging gently as the holder of the offending grinder squints in your direction. One of the older men, who you now realise was at least somewhat supervising the activity, steps forward and flicks his hands at you in a shooing motion, "Oi you, piss off. We've got our safety supervisor, we don't need your nose in our business."`
+			choice
+				`	(Challenge the claim.)`
+				`	(Leave them be.)`
+					go to leave
+			`	You point out that what you were seeing did not look at all safe, supervised or not, and the man glances around briefly before huffily making his way over to stand opposite you behind the fence. "Look stranger," he says, "work here is hard to find. For those working here, we don't have the time to spare. But the pay is good and we can at least feed our kids. Sure some people get a few scrapes and bruises but no one has died on my watch, and Rik only lost his smallest finger. If you feel the need to watch then fine, stand around, but leave us be ok. There ain't no one around here that'll care what you have to say anyway."`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:102]:
```suggestion
			`	You point out that what you were seeing did not look at all safe, supervised or not, and the man glances around briefly before huffily making his way over to stand opposite you behind the fence. "Look stranger," he says, "work here is hard to find. For those working here, we don't have the time to spare. But the pay is good and we can at least feed our kids. Sure, some people get a few scrapes and bruises but no one has died on my watch, and Rik only lost his smallest finger. If you feel the need to watch then fine, stand around, but leave us be. There ain't no one around here that'll care what you have to say anyway."`
```

File: data/human/culture conversations.txt [hunk 2]
```diff
@@ -80,11 +80,39 @@ outfit "Mug"
...
+			`	He looks you dead in the eyes and once he is sure you have understood his message he stumps back over to his crew which continue their work. You continue to watch them cut the dead ship apart until its pieces completely fill the trailer they have with them, before they proceed to haul the parts away to a factory in the distance.`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:103]:
```suggestion
			`	He looks you dead in the eyes, and once he is sure you understand his message he stomps back over to his crew, who continue their work. You watch them cut the dead ship apart until its pieces completely fill the trailer they have with them, before they proceed to haul the parts away to a factory in the distance.`
```

File: data/human/culture conversations.txt [hunk 3]
```diff
@@ -80,11 +80,39 @@ outfit "Mug"
...
+				decline
+			label leave
+			`	You turn and depart. A moment later you hear a clatter and a shout, but when you look back no one appears to be deceased.`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:106]:
```suggestion
			`	You turn and depart. A moment later you hear a clatter and a shout, but when you look back, no one appears to have died.`
```

File: data/human/culture conversations.txt [hunk 4]
```diff
@@ -163,6 +191,33 @@ mission "Class Trip to Port"
...
+mission "Blind Landing"
+	minor
+	landing
+	source
+		government "Free Worlds"
+		attributes "farming"
+		not attributes "station" "frontier"
+	to offer
+		random < 1
+		has "Trash Fire Crops:declined"
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:203]:
```suggestion
		has "Trash Fire Crops: declined"
```

File: data/human/culture conversations.txt [hunk 5]
```diff
@@ -163,6 +191,33 @@ mission "Class Trip to Port"
...
+	on offer
+		conversation
+			`Smog fills your windscreen as you make your descent, trying and failing to see anything more than a few metres in front of you. The autopilot systems show solid green across the full range of indicators though, and traffic control has confirmed a lock.`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:206]:
```suggestion
			`Smog fills your windscreen as you make your descend, trying and failing to see anything more than a few meters in front of you. The autopilot systems show solid green across the full range of indicators though, and traffic control has confirmed a lock.`
```

File: data/human/culture conversations.txt [hunk 6]
```diff
@@ -163,6 +191,33 @@ mission "Class Trip to Port"
...
+			choice
+				`	(Land on autopilot.)`
+				`	(Return to orbit.)`
+					goto orbit
+			`	You let the craft perform the landing, staring blankly at the haze until it clears.`
+			`	On the ground, you see the cause, fields upon fields of plantations burning. You see people arguing with firefighters, trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
+			`	You recall seeing something like this years ago, of farmers burning their crops to make the land fertile, and how bad the air was for months.`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:213]: But what if the player gets offered this mission right after the last one?

File: data/human/culture conversations.txt [hunk 7]
```diff
@@ -163,6 +191,33 @@ mission "Class Trip to Port"
...
+				decline
+			label orbit
+			`	Unwilling to trust landing blind into an unknown situation you return to orbit and rejoin the spaceport processing queue. One day of delay a worthwhile price to pay for surety.`
+				decline
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:217]:
```suggestion
				flee
```

File: data/human/culture conversations.txt [hunk 8]
```diff
@@ -163,6 +191,33 @@ mission "Class Trip to Port"
...
+			choice
+				`	(Land on autopilot.)`
+				`	(Return to orbit.)`
+					goto orbit
+			`	You let the craft perform the landing, staring blankly at the haze until it clears.`
+			`	On the ground, you see the cause, fields upon fields of plantations burning. You see people arguing with firefighters, trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
+			`	You recall seeing something like this years ago, of farmers burning their crops to make the land fertile, and how bad the air was for months.`
+				decline
+			label orbit
+			`	Unwilling to trust landing blind into an unknown situation you return to orbit and rejoin the spaceport processing queue. One day of delay a worthwhile price to pay for surety.`
```
Comment (Anarchist2) [inline data/human/culture conversations.txt:216]:
```suggestion
			`	Unwilling to trust landing blind into an unknown situation, you return to orbit and rejoin the spaceport processing queue. One day of delay is a worthwhile price to pay for surety.`
```

File: data/human/culture conversations.txt [hunk 9]
```diff
@@ -80,11 +80,39 @@ outfit "Mug"
...
+			`	Despite the distance and absent traffic though it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line which obstructs your path to anything interesting, beyond which you can see a throng of men cutting away at a rather well kept spaceship. Parts regularly fall to the floor below, apparently heedless of those working below.`
+			`	Even as you watch you see a hull plate swinging dangerously above a workers head.`
+			choice
+				`	(Call out a warning.)`
+				`	(Leave them be.)`
+					goto leave
+			`	You call out a warning and most of the men pause to look at you, the hull plate swinging gently as the holder of the offending grinder squints in your direction. One of the older men, who you now realise was at least somewhat supervising the activity, steps forward and flicks his hands at you in a shooing motion, "Oi you, piss off. We've got our safety supervisor, we don't need your nose in our business."`
```
[the remark this answers is not in the document]
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:97]: "at least somewhat" was definitely correct. As structure it infers that they were performing the role but not to the standard you would expect, or not well enough to have been indicated prior to intervention.

The rest of those corrections are rather valid though.

File: data/human/culture conversations.txt (hunk 1 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:102]:
```suggestion
`	You point out that what you were seeing did not look at all safe, supervised or not, and the man glances around briefly before huffily making his way over to stand opposite you behind the fence. "Look stranger," he says, "work here is hard to find. For those working here, we don't have the time to spare. But the pay is good and we can at least feed our kids. Sure, some people get a few scrapes and bruises but no one has died on my watch, and Rik only lost his smallest finger. If you feel the need to watch then fine, stand around, but leave us be okay. There ain't no one around here that'll care what you have to say anyway."`
```

File: data/human/culture conversations.txt (hunk 3 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:106]: Ehhh

File: data/human/culture conversations.txt (hunk 5 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:206]: It is not "descend"

File: data/human/culture conversations.txt (hunk 6 above)
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:213]: <1% chance?

Comment (Amazinite) [reply data/human/culture conversations.txt:213]: A 1% offer chance still leave room for this mission being offered close enough that "years ago" isn't an accurate descriptor. If you want to use language like this I'd say better safe than sorry and use an event with a 2 year delay. (If you want to get fancy with it you could instead do something like have `"bumper crop year" = "year"` in the `on offer` of the other mission and `"year" > "bumper crop year" + 2` in the `to offer` of this mission.)

File: data/human/culture conversations.txt (hunk 5 above)
Comment (Anarchist2) [reply data/human/culture conversations.txt:206]: Fixed.

File: data/human/culture conversations.txt (hunk 1 above)
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:102]: Would it not be "Leave us be, okay?"

File: data/human/culture conversations.txt [hunk 10]
```diff
@@ -80,11 +80,39 @@ outfit "Mug"
...
+mission "New Holand Foibles"
+	minor
+	source "New Holland"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning metal fills the air as you walk down the uncrowded streets of the factory town near the spaceport, keeping away from the arterial supply route directly feeding the port and its hangars.`
+			`	Despite the distance and absent traffic though it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line which obstructs your path to anything interesting, beyond which you can see a throng of men cutting away at a rather well kept spaceship. Parts regularly fall to the floor below, apparently heedless of those working below.`
+			`	Even as you watch you see a hull plate swinging dangerously above a workers head.`
```
[the remark this answers is not in the document]
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:92]: Nope. The first part adds to the sentence before. Without it, there's no real continuation of the emphasis.

File: data/human/culture conversations.txt (hunk 9 above)
[the remark this answers is not in the document]
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:97]:
```suggestion
			`	You call out a warning and most of the men pause to look at you, the hull plate swinging gently as the holder of the offending grinder squints in your direction. One of the older men, who you now realize was at least somewhat supervising the activity, steps forward and flicks his hands at you in a shooing motion. "Oi you, piss off. We've got our safety supervisor, we don't need your nose in our business."`
```

File: data/human/culture conversations.txt (hunk 1 above)
Comment (W1zrad) [reply data/human/culture conversations.txt:102]:
```suggestion
			`	You point out that what you were seeing did not look at all safe, supervised or not, and the man glances around briefly before huffily making his way over to stand opposite you behind the fence. "Look stranger," he says, "work here is hard to find. For those working here, we don't have the time to spare. But the pay is good and we can at least feed our kids. Sure, some people get a few scrapes and bruises, but no one has died on my watch, and Rik only lost his smallest finger. If you feel the need to watch then fine, stand around, but leave us be, okay? There ain't no one around here that'll care what you have to say anyway."`
```
Adding the comma reinforces how I think it would be said out loud, but personal preference.

File: data/human/culture conversations.txt (hunk 3 above)
Comment (W1zrad) [reply data/human/culture conversations.txt:106]:
```suggestion
			`	You turn and depart. A moment later you hear a clatter and a shout, but when you look back no one appears to have been injured.`
```
 How about this?

File: data/human/culture conversations.txt [hunk 11]
```diff
@@ -163,6 +192,34 @@ mission "Class Trip to Port"
...
+		has "Trash Fire Crops: declined"
+	on offer
+		conversation
+			`Smog fills your windscreen as you make your descent, trying and failing to see anything more than a few metres in front of you. The autopilot systems show solid green across the full range of indicators though, and traffic control has confirmed a lock.`
+			choice
+				`	(Land on autopilot.)`
+				`	(Return to orbit.)`
+					goto orbit
+			`	You let the craft perform the landing, staring blankly at the haze until it clears.`
+			`	On the ground, you see the cause, fields upon fields of plantations burning. You see people arguing with firefighters, trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
```
Comment (W1zrad) [inline data/human/culture conversations.txt:214]:
```suggestion
			`	On the ground, you see the cause: fields upon fields of plantations burning. You see people arguing with firefighters, trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
```
Not really a fan of the narrative here. As I see it, there are two possibilities:
1) This is planned and the folks would have proper permits for controlled burn. If so, then why are they having to argue with firefighters about putting it out?
2) The farmers just decided to do this without telling any government authority/the fire dept. If so, I think there's a much larger problem than just people arguing with firefighters. This seems like a pretty large hazard, and as you say below, it makes the air bad for months.
Also, if this is something happening every few years, why is this conflict happening at all? Surely the firefighters and farmers would be aware of what's going on instead of arguing to each other as the event goes down.
Basically, there's too little information here for me to reasonably explain what `<first>` is witnessing. A few extra sentences would be helpful.

File: data/human/culture conversations.txt [hunk 12]
```diff
@@ -293,3 +350,19 @@ mission "Tarazed Ship Christening"
...
+mission "Sojourn Sunday Sales"
+	minor
+	source
+		government "Pirate"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning gunpowder wafts about in the air as you enter a town square, watching as a body is rolled off of a raised platform. Cries and screams filled the air as a masked man drags a lady across the platform and onto the central dias, an old fashioned pistol to her head as he asks her something you can't make out. Thankfully, the man doesn't shoot, taking his gun off her temple before subsequently knocking her out with his weapon.`
```
Comment (W1zrad) [inline data/human/culture conversations.txt:365]:
```suggestion
			`The smell of burning gunpowder wafts about in the air as you enter a town square, watching as a body is rolled off of a raised platform. Cries and screams fill the air as a masked man drags a lady across the platform and onto the central dias, an old fashioned pistol to her head as he asks her something you can't make out. Thankfully, the man doesn't shoot, taking his gun off her temple before subsequently knocking her out with his weapon.`
```

File: data/human/culture conversations.txt [hunk 13]
```diff
@@ -293,3 +350,19 @@ mission "Tarazed Ship Christening"
...
+			`	Moments later the small crowd allows him to depart opposite your viewpoint without incident, and without explanation. You aren't quite sure what to make of this scene as the bodies are swiftly dragged away.`
```
Comment (W1zrad) [inline data/human/culture conversations.txt:366]:
```suggestion
			`	Moments later the small crowd allows him to depart opposite your viewpoint, without incident and without explanation. You aren't quite sure what to make of this scene as the bodies are swiftly dragged away.`
```

**Review by W1zrad — CHANGES_REQUESTED** [review]
Mostly minor, but got some problems with `Blind Landing`'s story.

File: data/human/culture conversations.txt (hunk 11 above)
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:214]: This one is based off the Haze situation in Singapore years ago. Back then, plantation workers in Indonesia would burn their crops for I don't remember reasons, and it caused visibility and air quality to drop for a good while.
I haven't seen the haze for years, but this story is intended to be told of illegal slash and burning.

Comment (W1zrad) [reply data/human/culture conversations.txt:214]: Gotcha. Then yeah, I think some elaboration would be good. I'm really not sure what the best direction would be to go with it, but I think the idea would be for there to be more going on between the farmers and firefighters, potentially involving authorities.

Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:214]: Maybe more people on the ground? I don't want to involve the player into the whole political situation and just an observer far away.

File: data/human/culture conversations.txt [hunk 14]
```diff
@@ -80,15 +80,44 @@ outfit "Mug"
...
 
+mission "New Holland Foibles"
+	minor
+	source "New Holland"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`The smell of burning metal fills the air as you walk down the uncrowded streets of the factory town near the spaceport, keeping away from the arterial supply route directly feeding the port and its hangars.`
+			`	Despite the distance and absent traffic though, it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line which obstructs your path to anything interesting. Beyond it, you can see a throng of men cutting away at a rather well kept spaceship. Parts regularly fall to the floor below, apparently heedless of those working below.`
```
[the remark this answers is not in the document]
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:91]: I don't overly like just "obstructs your path" - I in fact explicitly adapted from that in the original piece. It didn't make sense to me that you would walk down a street and just be blocked by a fenced open area.
"their companions" also feels unnecessary but eh

File: data/human/culture conversations.txt (hunk 11 above)
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:214]:
```suggestion
			`	On the ground, you see the cause, fields upon fields of plantations burning. You see a crowd blocking off a smaller group of firefighters, arguing and trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
```Test Case 1 of trying a rewrite.

File: data/human/culture conversations.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:91]: I agree that with both. Their companions is unnecessary and the rest just need approval.

[the remark this answers is not in the document]
Comment (W1zrad) [reply data/human/culture conversations.txt:91]:
```suggestion
			`	Despite the distance and absent traffic though, it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line. Beyond it, you can see a throng of men cutting away at a spaceship. Parts regularly fall to the floor below, the workers apparently heedless of those below.`
```
How about this? I might have misunderstood the scene and where the fence is oriented relative to the alley.

File: data/human/culture conversations.txt (hunk 11 above)
Comment (W1zrad) [reply data/human/culture conversations.txt:214]:
```suggestion
			`	On the ground, you see the cause, fields upon fields of plantations burning. You see a crowd of mostly farmers blocking off a smaller group of firefighters, arguing and trying to stop them from putting out the flames as planes fly overhead, dropping water and retardants wherever necessary.`
```
Maybe mentioning that the crowd is made of farmers helps clarify the burn's intentional. And I get where you're coming from: looking back at this a couple weeks later, it's a little unreasonable to try and explain the entire context to this situation within the story.

File: data/human/culture conversations.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:91]: Not entirely sure of the worker part. @MasterOfGrey?

File: data/human/culture conversations.txt (hunk 11 above)
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:214]: Yee, the crowd of farmers would be better. 
My alternative take would be to show that they're farmers than tell the reader they're what they are, but this is one of those points where I'd throw in the towel and let some future writer figure it out and edit.

File: data/human/culture conversations.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (MasterOfGrey) [reply data/human/culture conversations.txt:91]: >  I might have misunderstood the scene and where the fence is oriented relative to the alley.

The idea being you walk down the alley, and at some point it is no longer hemmed in by buildings on either side. It is instead just fenced on one side. The fence blocks your access to the things next to the otherwise unremarkable alley.

[the remark this answers is not in the document]
Comment (W1zrad) [reply data/human/culture conversations.txt:91]: Gotcha, makes sense to me.

> Not entirely sure of the worker part.

That's to fix what I mentioned above:

>My main problem is just as it was, apparently heedless seemed to refer to the ship parts themselves, which are inanimate objects.

Including `the workers` makes it clear it's the workers who are heedless, not the ship parts.

[the remark this answers is not in the document]
Comment (TheMarksman-ES) [reply data/human/culture conversations.txt:91]: It works. Just need to edit out the repeated "below" and it _should_ be fine.

[the remark this answers is not in the document]
Comment (W1zrad) [reply data/human/culture conversations.txt:91]: Oops, I gotcha.
```suggestion
			`	Despite the distance and absent traffic though, it is far from quiet. Several factory buildings make discernible thrumming noises, and grinders echo amidst the silence as you investigate. Turning a corner, you come across an alley running along a lengthy chain link fence line. Beyond it, you can see a throng of men cutting away at a spaceship. Parts regularly fall to the floor, the workers apparently heedless of those below.`
```
@MasterOfGrey look good to you as well?

**Review by W1zrad — APPROVED** [review]
